R/assertions.R

```diff
@@ -12,18 +12,14 @@ on_failure(is_input_id) <- function(call, env) {
 
 
 is_header <- function(x) {
-  is.null(x) || (is.character(x) && length(x) == 1 && !is.na(x))
+  is.na(x) || (is.character(x) && length(x) == 1)
 }
 
 on_failure(is_header) <- function(call, env) {
```

Comment:
IDK how this worked before

```suggestion
on_failure <- function(call, env) {
```